newm-server/src/main/kotlin/io/newm/server/features/distribution/eveara/EvearaDistributionRepositoryImpl.kt

```diff
@@ -1359,84 +1358,40 @@ class EvearaDistributionRepositoryImpl(
                         log.info { "Updated collab distribution artist ${collabUser.email} with id ${response.artistData?.artistId}: ${response.message}" }
                     }
                 } ?: run {
-                    val artist = getArtists(collabUser).artists.firstOrNull()
```

Comment:
This line here is the bug. We're getting artists from eveara for a user that is *NOT* the current user distributing a track. We then get/copy the distributionArtistId underneath the collab user when what we need to do is create a new distributionArtistId under THIS current user.